slam-libraries/viam-cartographer/src/slam_service/config.cc

```diff
@@ -36,9 +40,6 @@ int ParseAndValidateConfigParams(int argc, char** argv) {
     } else if (FLAGS_port.empty()) {
         LOG(ERROR) << "-port is missing.\n";
         return EXIT_FAILURE;
-    } else if (FLAGS_sensors.empty()) {
-        LOG(ERROR) << "-sensors is missing.\n";
-        // return EXIT_FAILURE;
     }
```

Comment:
[q] Why did you remove this?

Reply:
This error was shown if sensors was empty, which is expected for offline.